### PR TITLE
Subscribe & Author Muting include Meta Data Link

### PR DESCRIPTION
**Pull request by attilaher**

### CODE REVIEW

File: www/js/controllers/favorite.js [hunk 1]
```diff
@@ -0,0 +1,423 @@
+angular.module('app.controllers').controller('favorite', function ($scope, $timeout, socialActivity, homeCtrlParams,
+        profile, favorite, groups, flurry, $ionicScrollDelegate) {
+
+  flurry.log('news feed');
```
Comment (austinpapp) [inline www/js/controllers/favorite.js:4]: wouldn't this be the "favorites" controller? not the news feed?

Comment (attilaher) [inline www/js/controllers/favorite.js:4]: This is "favorite" controller.

Comment (austinpapp) [inline www/js/controllers/favorite.js:4]: sorry - that wasn't very clear (after rereading). The log says "news feed". should it say "favorites" ?

**Comment by austinpapp** [comment]
This pull request handles multiple issues
1. closes #19 
2. closes #20
3. closes #21
4. closes #22
5. closes #13

**Pull request merged**